features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/impl/internal/DatadogScopeListenerAdapter.kt

```diff
@@ -5,10 +5,13 @@
  */
 package com.datadog.android.trace.impl.internal
 
+import androidx.annotation.VisibleForTesting
 import com.datadog.android.trace.api.scope.DataScopeListener
 import com.datadog.trace.api.scopemanager.ScopeListener
 
-internal class DatadogScopeListenerAdapter(private val delegate: DataScopeListener) : ScopeListener {
+internal class DatadogScopeListenerAdapter(
+    @get:VisibleForTesting internal val delegate: DataScopeListener
```

Comment:
minor: not sure if `VisibleForTesting` annotation is useful for Kotlin code. It is different for Java, where it is usually applied to the `public` members and is usually and indicator that such member still not a part of official API, but here it is internal anyway.

Reply:
I put `@VisibleForTesting` annotation in such places as an explanation why this property should be `internal` instead of private.  Like here it's means that it's internal only for the testing purposes.